src/Producer/Serializer/ProductSerializer.php

```diff
@@ -110,6 +119,21 @@ protected function getTranslations(ProductInterface $product): array
         return $translations;
     }
 
+    private function getAttributeTranslations(AttributeInterface $attribute): array
+    {
+        $translations = [];
+        /** @var AttributeTranslationInterface $translation */
+        foreach ($attribute->getTranslations() as $translation) {
+            $translations[] = [
```

Comment:
I guess `$translations[$locale]` would be more convenient for the consumer bundle

Reply:
the consumer will convert that to the payload like this

Reply:
see https://github.com/sulu/SuluSyliusConsumerBundle/pull/122/files#diff-7482afb514ac442d341c763fd24f45d9c4b31